azure/functions_worker/dispatcher.py

```diff
@@ -374,6 +375,11 @@ def _serialize_exception(self, exc):
             for var in env_vars:
                 os.environ[var] = env_vars[var]
 
+            importlib.invalidate_caches()
```

Comment:
Please add a comment on why this is here (e.g to repopulate `azure.__path__` on worker specialization).

Reply:
Since we are only specializing the azure package, I don't think this line of code is necessary anymore. Thanks for picking this out.

Reply:
Moving to the main repo (Fixed nested namespace library import issue #492)